M2/Macaulay2/packages/Macaulay2Doc/overview_packages.m2

```diff
@@ -37,17 +37,19 @@ Node
     "authors of Macaulay2 packages"
   Description
     Text
-      Here is a list of people who have authored packages that are distributed with Macaulay2.
+      The following people have authored packages that are distributed with Macaulay2:
     Code
-      authors := new HashTable from flatten apply(separate_" " version#"packages", pkgname -> (
+      authors := new MutableHashTable from flatten apply(separate_" " version#"packages", pkgname -> (
               pkgopts := readPackage pkgname;
               apply(pkgopts.Authors, author -> (
                       author = new OptionTable from author;
                       if author.?Name and (not authors#?(author.Name) or not instance(authors#(author.Name), HREF))
                       then author.Name => if author.?HomePage then HREF {author.HomePage, author.Name} else author.Name))));
+      scan(select(keys authors, match_{"(C|c)ontribut", "(M|m)aintain", "(A|a)uthor", "(T|t)hank", "(Michael|Mike) Stillman"}),
```

Comment:
It looks like you're removing Mike as author from all the packages he's an author of.

Reply:
No, "Michael E. Stillman" is still listed.

Reply:
What's the purpose of matching on Stillman, then?

Reply:
I don't want to exclude all other Mikes or Michaels. Is there a problem here?

Reply:
What is the purpose of removing authors matching "(Michael|Mike) Stillman" from the list of authors?

Reply:
Clearly we don't want Mike's name appearing in three forms in the list of authors. This removes two of them so the list of authors is not redundant. Of all the changes in this PR, is this the only thing you have questions about?

Reply:
Wouldn't it be more uniform if you would edit the occurrences of his name where he appears as an author, the way you do for everyone else?  As here:
```m2
diff --git a/M2/Macaulay2/packages/BGG.m2 b/M2/Macaulay2/packages/BGG.m2
index 28ec23e28..af0131a63 100644
--- a/M2/Macaulay2/packages/BGG.m2
+++ b/M2/Macaulay2/packages/BGG.m2
@@ -6,11 +6,12 @@ newPackage(
 	     {Name => "Hirotachi Abo", Email => "abo@uidaho.edu", HomePage => "http://www.webpages.uidaho.edu/~abo/"},
 	     {Name => "Wolfram Decker", Email => "decker@math.uni-sb.de", HomePage => "http://www.math.uni-sb.de/ag/decker/"},
 	     {Name => "David Eisenbud", Email => "de@msri.org", HomePage => "http://www.msri.org/~de/"},	     
-	     {Name => "Frank Schreyer", Email => "schreyer@math.uni-sb.de", HomePage => "http://www.math.uni-sb.de/ag/schreyer/"},
+	     {Name => "Frank-Olaf Schreyer", Email => "schreyer@math.uni-sb.de", HomePage => "http://www.math.uni-sb.de/ag/schreyer/"},
 	     {Name => "Gregory G. Smith", Email => "ggsmith@mast.queensu.ca", HomePage => "http://www.mast.queensu.ca/~ggsmith/"},
 	     {Name => "Mike Stillman", Email => "mike@math.cornell.edu", HomePage => "http://www.math.cornell.edu/~mike/"}
```

Reply:
This is a temporary patch, not a uniform change. Mike's name appears in 57 packages and he didn't have a clear preference to change them all into. If you have time, feel free to change them all when he has decided, but I wanted to get it done quickly and get to more important things.

Reply:
Oh, okay.   Meanwhile, the list of authors doesn't have all the information that the one I created manually does, so #23 should not be closed.  Compare:
--
<img width="1341" alt="Screen Shot 2020-10-08 at 8 51 41 AM" src="https://user-images.githubusercontent.com/700228/95468034-bdff0e00-0943-11eb-8a96-376012ea6ad9.png">
--
<img width="1003" alt="Screen Shot 2020-10-08 at 8 51 26 AM" src="https://user-images.githubusercontent.com/700228/95468042-bf303b00-0943-11eb-82bd-52f28617dda3.png">
--
Is that on your radar eventually?

Reply:
> Is that on your radar eventually?

No. I think that page should be removed because it excludes many many many contributors. The versions are too old and many links and emails don't even work anymore either.

I tried listing packages that people have contributed to, but this is not feasible anymore when there are ~200 packages. Some packages are repeated many times and some names will have way too many packages listed in front of them (e.g. Mike is listed in 57 packages!)

Reply:
> This is a temporary patch, not a uniform change. Mike's name appears in 57 packages and he didn't have a clear preference to change them all into. If you have time, feel free to change them all when he has decided, but I wanted to get it done quickly and get to more important things.

I do want to have my name appear in only one way.  Perhaps "Michael Stillman"?  If you want, I can go change them all.